### PR TITLE
Overall size

### PR DESCRIPTION
**Pull request by guacamole**
### Motivation & Context:

Sidebar and background were both light. The pages lacked contrast and the sidebar was too big when zoomed in.
The home page and how does it work sections were also pretty plain with slate background and I feel like it needed a bit more drama. 
This PR fixes  the Sidebar by making it darker and reducing the overall size. Along with this, the mesh has been replaced with a beautiful gradient which adds just enough drama to the hero section

### Description:

- Worked on Sidebar colors and overall sizes of font and icons
- Removed the mesh and added a radial gradient to body

### Types of Changes:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### PR Checklist:

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I've read the CONTRIBUTION guide
- [x] I have signed-off my commits with git commit -s

### CODE REVIEW

File: src/routes/(marketing)/about/+page.svelte
Comment (jay-dee7) [inline src/routes/(marketing)/about/+page.svelte:1]: @guacamole Github & Twitter links aren't working on about page - 
image

Comment (guacamole) [reply src/routes/(marketing)/about/+page.svelte:1]: I'm on it!

**Review by jay-dee7 — COMMENTED** [review]
Other than the links, all seems to pitch perfect :)

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**